diff --git ncurses-clean/aclocal.m4 ncurses-workdir/aclocal.m4
index 1463b24..f17b19c 100644
--- ncurses-clean/aclocal.m4
+++ ncurses-workdir/aclocal.m4
@@ -7843,7 +7843,7 @@ CF_EOF
 		fi
 		cf_cv_rm_so_locs=yes
 		;;
-	(linux*|gnu*|k*bsd*-gnu)
+	(roxy*|linux*|gnu*|k*bsd*-gnu)
 		if test "$DFT_LWR_MODEL" = "shared" && test -n "$LD_RPATH_OPT" ; then
 			LOCAL_LDFLAGS="${LD_RPATH_OPT}\$(LOCAL_LIBDIR)"
 			LOCAL_LDFLAGS2="$LOCAL_LDFLAGS"
diff --git ncurses-clean/configure ncurses-workdir/configure
index 48ad381..787bc01 100755
--- ncurses-clean/configure
+++ ncurses-workdir/configure
@@ -7123,7 +7123,7 @@ echo "${ECHO_T}$cf_cv_ldflags_search_paths_first" >&6
 		fi
 		cf_cv_rm_so_locs=yes
 		;;
-	(linux*|gnu*|k*bsd*-gnu)
+	(roxy*|linux*|gnu*|k*bsd*-gnu)
 		if test "$DFT_LWR_MODEL" = "shared" && test -n "$LD_RPATH_OPT" ; then
 			LOCAL_LDFLAGS="${LD_RPATH_OPT}\$(LOCAL_LIBDIR)"
 			LOCAL_LDFLAGS2="$LOCAL_LDFLAGS"
