diff --git xf86-video-fbdev-clean/configure xf86-video-fbdev-workdir/configure
index 55af631..c6102bd 100755
--- xf86-video-fbdev-clean/configure
+++ xf86-video-fbdev-workdir/configure
@@ -658,6 +658,7 @@ am__EXEEXT_TRUE
 LTLIBOBJS
 LIBOBJS
 DRIVER_NAME
+SYSROOT
 moduledir
 PCIACCESS_LIBS
 PCIACCESS_CFLAGS
diff --git xf86-video-fbdev-clean/configure.ac xf86-video-fbdev-workdir/configure.ac
index 19fa1c3..27aab06 100644
--- xf86-video-fbdev-clean/configure.ac
+++ xf86-video-fbdev-workdir/configure.ac
@@ -77,6 +77,8 @@ AC_SUBST([moduledir])
 DRIVER_NAME=fbdev
 AC_SUBST([DRIVER_NAME])
 
+AC_SUBST([SYSROOT])
+
 AC_CONFIG_FILES([
                 Makefile
                 src/Makefile
diff --git xf86-video-fbdev-clean/src/Makefile.am xf86-video-fbdev-workdir/src/Makefile.am
index cc0290f..2bae4e8 100644
--- xf86-video-fbdev-clean/src/Makefile.am
+++ xf86-video-fbdev-workdir/src/Makefile.am
@@ -25,7 +25,7 @@
 # TODO: -nostdlib/-Bstatic/-lgcc platform magic, not installing the .a, etc.
 AM_CFLAGS = $(BASE_CFLAGS) $(XORG_CFLAGS)
 fbdev_drv_la_LTLIBRARIES = fbdev_drv.la
-fbdev_drv_la_LDFLAGS = -module -avoid-version
+fbdev_drv_la_LDFLAGS = -module -avoid-version -R@moduledir@ -L@SYSROOT@@moduledir@ -lfbdevhw -lshadow
 fbdev_drv_ladir = @moduledir@/drivers
 
 fbdev_drv_la_SOURCES = \
diff --git xf86-video-fbdev-clean/src/Makefile.in xf86-video-fbdev-workdir/src/Makefile.in
index 016d61d..ef52fe9 100644
--- xf86-video-fbdev-clean/src/Makefile.in
+++ xf86-video-fbdev-workdir/src/Makefile.in
@@ -307,6 +307,7 @@ SET_MAKE = @SET_MAKE@
 SHELL = @SHELL@
 STRICT_CFLAGS = @STRICT_CFLAGS@
 STRIP = @STRIP@
+SYSROOT = @SYSROOT@
 VERSION = @VERSION@
 XORG_CFLAGS = @XORG_CFLAGS@
 XORG_LIBS = @XORG_LIBS@
@@ -373,7 +374,7 @@ top_srcdir = @top_srcdir@
 # TODO: -nostdlib/-Bstatic/-lgcc platform magic, not installing the .a, etc.
 AM_CFLAGS = $(BASE_CFLAGS) $(XORG_CFLAGS)
 fbdev_drv_la_LTLIBRARIES = fbdev_drv.la
-fbdev_drv_la_LDFLAGS = -module -avoid-version
+fbdev_drv_la_LDFLAGS = -module -avoid-version -R@moduledir@ -L@SYSROOT@@moduledir@ -lfbdevhw -lshadow
 fbdev_drv_ladir = @moduledir@/drivers
 fbdev_drv_la_SOURCES = \
          fbdev.c
